src/Project02.py

```diff
@@ -1,202 +1,197 @@
-            if lst[i+1]=="2":
-                date_m = pd.to_datetime('-'.join(lst[i+3:i+6]))
-                families.Married[idx] = date_m.strftime("%b-%d-%Y")
-        if "_CURRENT" in lst:
-            div_case = lst.index("_CURRENT")
-            if lst[div_case+1] == "N":
-                families.Divorced[idx] = "True"
-            else:
-                families.Divorced[idx] = "False"
-        
-        if 'HUSB' in lst:
-            i = lst.index('HUSB')
-            families['Husband ID'][idx] = lst[i+1]
-            families['Husband Name'][idx] = list(individuals.Name[individuals.ID == lst[i+1]])[0]
-            
-        if 'WIFE' in lst:
-            i = lst.index('WIFE')
-            families['Wife ID'][idx] = lst[i+1]
-            families['Wife Name'][idx] = list(individuals.Name[individuals.ID == lst[i+1]])[0]
-        if "CHIL" in lst:
-            chil_ids = [idx for idx, val in enumerate(lst) if val in lst[:idx] and val == "CHIL"]
-            chil_ids = [lst.index("CHIL")] + chil_ids
-            for n in range(len(chil_ids)):
-                chil_ids[n] += 1
-                chil_ids[n] = lst[chil_ids[n]]
-            families.Children[idx] = chil_ids
-
-    return families
-
-
-def displayTable(gedcom_name):
-    individuals = createIndividualsDataFrame(gedcom_name)
-    families = createFamiliesDataFrame(gedcom_name)
-
-    otp = open("Project03_Output.txt", "w")
-    otp.truncate(0)
-    otp.write("Individuals: \n")
-    otp.write(tabulate(individuals, headers='keys', tablefmt='psql'))
-    otp.write("\n")
-    otp.write("Families: \n")
-    otp.write(str(families))
-    otp.close()
-
-def displayOutput(gedcom_name):
-    supported = ['INDI', '0 NOTE', '0 HEAD', '0 TRLR', 'FAM', '1 NAME', '1 SEX', '1 BIRT', '1 DEAT', '1 FAMC', 
-                 '1 FAMS', '1 MARR', '1 HUSB', '1 WIFE', '1 CHIL', '1 DIV', '2 DATE']
-
-    output_lines = []
-    input_lines = collectInputFile(gedcom_name)
-    for line in input_lines:
-        if any(s in line for s in supported):
-            t = line.strip().split()
-            if 'INDI' in t:
-                idx = t.index('INDI')
-                output_lines.append('|'.join([t[0]] + [t[idx]] + ['Y'] + [' '.join(t[1:idx])]))
-            elif 'FAM' in t:
-                idx = t.index('FAM')
-                output_lines.append('|'.join([t[0]] + [t[idx]] + ['Y'] + [' '.join(t[1:idx])]))
-            else:
-                output_lines.append('|'.join(t[0:2] + ['Y'] + [' '.join(t[2:])]))
-        else:
-            t = line.strip().split()
-            output_lines.append('|'.join(t[0:2] + ['N'] + [' '.join(t[2:])]))
-
-
-    otp = open("outputProject02.txt", "w")
-    for n in range(len(input_lines)):
-        otp.write("\n" + "--> " + input_lines[n])
-        otp.write("\n" + "<-- " + output_lines[n] + "\n")
-    otp.close()
-
-if __name__ == '__main__':
-    file_name = input("Enter the name of the GEDCOM file: ")
-    displayTable(file_name)
-
-
-
-
-
+
```

Comment:
just remove the extra white space. you could use git add -p to view what changes your staging to commit before you actually commit them. Should help to avoid committing these unneeded change like this